minio/minio
# Capture TLS info in health report

## Description

Capture TLS info in health report so that TLS related checks
can be added in health-analyzer.

## Motivation and Context

Important checks related to TLS configuration can be added in
health-analyzer if this data is available in the health report e.g.

- TLS enabled or not
- Certificates expired
- Non-optimal TLS configuration

## How to test this PR?

- Build `minio` with this PR
- Start the minio server with the newly built binary and create a new cluster
- Build `mc` compiling it with latest `main` branch of `madmin-go`
- Use this newly built binary for subsequent steps
- Add the newly created cluster config using `mc config host add ...`
- Generate the health report for this new cluster using `mc admin subnet health {alias} --airgap`
- Verify that a new node `minio -> info -> tls` is present in the report

## Types of changes

## Checklist:
